pyaedt/edb_core/layout.py

```diff
@@ -259,12 +259,16 @@ def get_polygon_points(self, polygon):
         Parameters
         ----------
         polygon :
-            Name of the polygon.
+            class: `pyaedt.edb_core.edb_data.primitives_data.EDBPrimitives`
```

Comment:
@svandenb-dev Shouldn't the class be used as the data type? I've not see anything formatted like this before.